ee/controllers/phdeployment_controller.go

```diff
@@ -708,6 +711,65 @@ func (r *PhDeploymentReconciler) buildDeployment(ctx context.Context, phDeployme
 	tolerations := make([]corev1.Toleration, 0)
 	tolerations = append(tolerations, spawner.Tolerations...)
 
+	initContainers := make([]corev1.Container, 0)
+	volumes := []corev1.Volume{
+		{
+			Name: "podinfo",
+			VolumeSource: corev1.VolumeSource{
+				DownwardAPI: &corev1.DownwardAPIVolumeSource{
+					Items: []corev1.DownwardAPIVolumeFile{
+						{
+							Path:     "annotations",
+							FieldRef: &corev1.ObjectFieldSelector{FieldPath: "metadata.annotations", APIVersion: "v1"},
+						},
+					},
+					DefaultMode: &defaultMode,
+				},
+			},
+		},
+	}
+	if len(phDeployment.Spec.Predictors[0].ModelURI) > 0 {
+		modelURI := phDeployment.Spec.Predictors[0].ModelURI
+		initContainersVolumeMount := []corev1.VolumeMount{
+			corev1.VolumeMount{
+				Name:      "model-storage",
+				MountPath: "/mnt/models",
+			},
+		}
+
+		if len(modelURI) > 4 && strings.HasPrefix(modelURI, "phfs") {
```

Comment:
ok! In line 744 now.

Reply:
Sorry. I meant if the PHFS is enabled (phfsPVC is not empty) rather than checking the existence of pvc.  

Currently, the controller has no PVC get permission.

Reply:
got it. I fix it, thx!